deploy/dev-ngsa-asb-eastus/ngsa/dev/ngsa/ngsa-memory.yaml

```diff
@@ -33,6 +33,8 @@ spec:
           - az-eastus-dev
           - --region
           - eastus
+          - --cosmos-auth-type
```

Comment:
Let AutoGitOps update the deployment yaml files. In this case, the cosmos-auth-type flag is not needed because ngsa-memory uses an in-memory db